Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -365,55 +368,177 @@ def start_chrome_headless(chrome_port, chrome_binary=CHROME_EXE, user_options=""
     return None, None
 
 
-def terminate_chrome(browser):
-    global CHROME_PROCESS
-    demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
+def terminate_chrome(chrome_port='', killall=False):
+    processes = subprocess.check_output(['ps', 'auxww'], stderr=subprocess.STDOUT, text=True).splitlines()
+    chrome_renderer_identifiers = ["--type=renderer"]
+    chrome_identifiers = ["chrom", "headless", f"--remote-debugging-port={chrome_port}"]
+    process_in_list = [process for process in processes
+                       if all(identifier in process for identifier in chrome_identifiers)
+                       and not any(identifier in process for identifier in chrome_renderer_identifiers)]
 
-    threading.excepthook = excepthook_recv_loop
+    if killall:
+        pids = [int(process.split()[1]) for process in process_in_list]
+    else:
+        process_string_representation = process_in_list[0]
+        pids = [int(process_string_representation.split()[1])]
```

Comment:
Lets add some comments here for better readability to explain what exactly is done in here.
Also, please add docstring to the method.
Why do we take only the first process if not killall? we for sure will have only one?

Reply:
If not killall, by the logic i call the function will be for sure valid chrome_port to terminate.
ill search this port and there will be only one process that matches for it.
because of the code above that should be generic for killall (with many processes) as well so we get the processes as a list, then, in this case, the list contains just one process.